packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -174,6 +179,25 @@ function transform(codeGen: CodeGen): t.Expression {
         }
     }
 
+    function transformScopedSlotContent(scopedSlotContent: ScopedSlotContent): t.Expression {
+        const {
+            slotName,
+            directive: { value: dataIdentifier },
+        } = scopedSlotContent;
+        codeGen.beginScope();
+        codeGen.declareIdentifier(dataIdentifier);
+        // TODO [#9999]: Next Step: Return a VFragment
```

Comment:
🙃

Reply:
This TODO is still missing an issue number.